src/values/ObjectValue.js

```diff
@@ -280,6 +282,14 @@ export default class ObjectValue extends ConcreteValue {
   // ES2015 classes
   $IsClassPrototype: boolean;
 
+  // We track some internal state as properties on the global object, these should
+  // never be serialized.
+  refuseSerialization: boolean;
+
+  addtemporalAlias(temporalValue: AbstractObjectValue) {
+    this.temporalAlias = temporalValue;
```

Comment:
Is it okay to call this multiple times? Maybe add precondition that `this.temporalAlias===undefined`.

Reply:
It is OK to call it multiple times since you want to update it as time goes by.